src/glfs-cli-commands.c

```diff
@@ -154,9 +174,8 @@ handle_quit (struct cli_context *ctx)
 {
         cli_disconnect (ctx);
 
-        if (ctx->argv) {
+        if (ctx->argv)
```

Comment:
The decision to wrap one line conditional bodies with curly braces was intentional. Unless the gluster community feels strongly about changing this, I'd rather keep it that way.